### ISSUE TITLE
There is a line of code that is wrong.

### ISSUE THREAD

**Issue by alrisgn** [princeton-vl/RAFT#192]
In corr.py, there is a line of code is:
delta = torch.stack(torch.meshgrid(dy, dx), axis=-1)

Shouldn't it be:
delta = torch.stack(torch.meshgrid(dx, dy, index="xy"), axis=-1)
?

Because the function "def bilinear_sampler(img, coords, mode='bilinear', mask=False)" looks the first place of the last dimension as x, and it looks the second place as y.

Am I wrong?

I'm confused, thanks very much for replying.

**Comment by baolitianxin** [comment]
Hello,may I ask if your issue has been resolved? Ialso have the same doubts

**Comment by alrisgn** [comment]
> Hello,may I ask if your issue has been resolved? Ialso have the same doubts

Sorry, I just saw it. I think it's true that the author made a mistake here, but it doesn't affect the effect, so there's no need to change it.

**Comment by alrisgn** [comment]
By the way, I noticed that the residual block in this code differs from the standard implementation in Kaiming He's original ResNet (it has an extra ReLU before the addition). Was this an intentional design choice or an oversight? Does this modification have a significant impact on performance in this specific context?
